### PR TITLE
add vikingdb and embedding

### PR DESCRIPTION
**Pull request by 19374242**

### CODE REVIEW

**Comment by chyroc** [comment]
Duplicated with https://github.com/langchain-ai/langchain/pull/14693

**Comment by 19374242** [comment]
> Duplicated with #14693

Then I will remove the embeddings part from my code

**Comment by 19374242** [comment]
> Duplicated with #14693

This is my first time participating in the Langchain project. I would like to inquire about when the code can be merged.

File: libs/langchain/langchain/vectorstores/vikingdb.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from langchain_community.vectorstores.vikngdb import VikingDBConfig, VikingDB
```
Comment (hwchase17) [inline libs/langchain/langchain/vectorstores/vikingdb.py:1]: no need to update langchain package

**Review by hwchase17 — COMMENTED** [review]
would be good to add notebook for example

File: libs/langchain/langchain/vectorstores/vikingdb.py (hunk 1 above)
Comment (19374242) [reply libs/langchain/langchain/vectorstores/vikingdb.py]: fixed

**Review by hwchase17 — APPROVED** [review]
lgtm, some sort of docs would be nice

**Comment by 19374242** [comment]
> lgtm, some sort of docs would be nice

I would like to ask how to resolve the error: "Error: langchain_community/vectorstores/vikngdb.py:1:1: I001 Import block is un-sorted or un-formatted." The first line of my code is "import logging."

**Comment by 19374242** [comment]
fixed

**Comment by 19374242** [comment]
> lgtm, some sort of docs would be nice

 I hope to merge the code first, and I will follow up with the documentation later.

**Comment by 19374242** [comment]
> lgtm, some sort of docs would be nice

Sorry to bother you, could you please review and merge my code? Thanks!

**Comment by 19374242** [comment]
Hello, could you please review and merge my code? Thank you very much. @hwchase17 @chyroc

**Pull request merged**